searx/templates/simple/messages/no_results.html

```diff
@@ -1,11 +1,28 @@
 {% from 'simple/icons.html' import icon_big %}
-<div class="dialog-error-block" role="alert">
-  <p><strong>{{ _('Sorry!') }}</strong></p>
-  <p>{{ _("No results were found. You can try to:") }}</p>
+<div class="dialog-error-block" role="alert">{{- '' -}}
+  <p>{{- '' -}}
+    <strong>
+      {%- if pageno == 1 -%}
+        {{ _('Sorry!') }}
+      {%- endif -%}
+    </strong>{{- '' -}}
+  </p>{{- '' -}}
+  <p>
+    {%- if pageno == 1 -%}
+      {{ _("No results were found. You can try to:") }}
+    {%- else -%}
+      {{ _("There are no more results.") }}
```

Comment:
You forgot the "You can try to: " part :).
```suggestion
      {{ _("There are no more results. You can try to:") }}
```

Reply:
> You forgot the "You can try to: " part :).

:facepalm: .. thanks for pointing out .. fixed in https://github.com/searxng/searxng/compare/ee0caf313079cb94c40808c50284a2765b77b624..875d83674d1d17b2996b7e7c46c9c3eb9afa0602